commenting_system/tests.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+from django.contrib.auth.models import User
+from commenting_system.models import Comment
+from Post.models import Post
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def user_list():
+    teardown_user_list()
+    username = 'Test-user{}'
+    email = '{}@gmail.com'
+    password = '{}password'
+    num_of_users = 5
+    return [User.objects.create_user(username.format(ind),
+                                     email.format("test-" + str(ind)),
+                                     password.format("test-" + str(ind)))
+            for ind in range(1, num_of_users + 1)]
```

Comment:
consider using this:
for _ in range (num_of_users):

Reply:
@Yarboa The only problem with that is that I will get numbers from 0 to (num_of_users - 1) instead of 1 to num_of_users

Reply:
Why cant you start from 0? it is a users list,

Reply:
Just not what I intended since I wanted it to be a serial number but it's only a personal preference, you just mean simplicity is more important at this part?

Reply:
\0/ correct